### PR TITLE
No duplicate downloads

### PR DESCRIPTION
**Pull request by Septias**

### CODE REVIEW

File: frontend/src/entry-points/shop.tsx [hunk 1]
```diff
@@ -67,13 +67,13 @@ function AppInfoModal(item: AppInfoWithState, onDownload: () => void, onForward:
...
+            <div class="send-button bg-gray-500">
               <div class="i-eos-icons:three-dots-loading text-white"></div>
-            </button>
+            </div>
           </Match><Match when={item.state === AppState.DownloadCancelled}>
-            <button class="send-button bg-red-500" >
+            <div class="send-button bg-red-500" >
               <div class="i-material-symbols:error text-white"></div>
-            </button>
+            </div>
```
Comment (hpk42) [inline frontend/src/entry-points/shop.tsx:76]: how is this preventing duplicate downloads?

Comment (Septias) [reply frontend/src/entry-points/shop.tsx:76]: onClick handler removed

**Comment by adbenitez** [comment]
> The file is downloaded twice in the background invisibly to the user? `sendToChat` is called twice?
 
sendToChat() is no longer triggered after receiving the file, so it is only about sending several download requests by spamming the download button

**Review by adbenitez — APPROVED** [review]
was not able to deploy it due to other issues but the code looks fine at a first glance

**Pull request merged**